packages/ai-jsx/src/batteries/sidekick/platform/conversation.tsx

```diff
@@ -68,6 +69,28 @@ export function present(conversationElement: ConversationMessage, outputFormat:
   return conversationElement.element;
 }
 
+/**
+ * Interjects content if the first conversation message in the children is a function call.
+ */
+function InterjectBeforeFunctionCall(
+  { interjection, children }: { interjection: AI.Node; children: AI.Node },
+  { memo }: AI.ComponentContext
+) {
+  const memoizedInterjection = memo(interjection);
+  return (
+    <ShowConversation
+      present={(message, index) => {
+        if (index === 0 && message.type === 'functionCall') {
```

Comment:
So we only interject if we see a `functionCall` in response to a `functionResponse`?
Or am I misunderstanding something?

Reply:
We interject before the `functionCall` (which in turn precedes the `functionResponse`).